RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -374,13 +375,17 @@ protected void executeRun(ExecutionContext context, boolean preventLoggingOnQuer
                     ExecutionService.getInstance().writeFilesForExecution(context);
                     boolean execute = true;
                     if (context.getExecutionContextLevel().isOrWasAllowedToSubmitJobs) { // Only do these checks, if we are not in query mode!
-                        List<String> invalidPreparedFiles = ExecutionService.getInstance().checkFilesPreparedForExecution(context);
+                        List<String> invalidPreparedFiles = ExecutionService.getInstance().checkForInaccessiblePreparedFiles(context);
                         boolean copiedAnalysisToolsAreExecutable = ExecutionService.getInstance().checkCopiedAnalysisTools(context);
-                        execute &= invalidPreparedFiles.size() == 0 && copiedAnalysisToolsAreExecutable;
-                        if (!execute) {
+                        boolean ignoreFileChecks = Roddy.isStrictModeEnabled() && Roddy.getCommandLineCall().isOptionSet(RoddyStartupOptions.ignorepreparedfilechecks);
```

Comment:
With more frequent use of feature toggles this may be useful:
```
boolean isStrictWithFeature(RoddyStartupOptions opt) {
   Roddy.isStrictModeEnabled() && Roddy.getCommandLineCall().isOptionSet(otp)
}
```